### ISSUE TITLE
(Windows) File lock on unsupported filetype

### ISSUE THREAD

**Issue by savchenko** [Lymphatus/caesium-image-compressor#213]
**Describe the bug**
Caesium locks file for writing.

**Software version**
v2.4.0

**Operating System information**
 - OS: Windows
 - Version 21H2

**To Reproduce**
1. Drag'n'drop TIFF file into the main window
2. Observe warning that filetype is not supported
3. Try to delete the original file

**Expected behavior**
The file remains available for writing.

**Screenshots**
N/A

**Application Log**
```
[2023-06-21 00:18:54.424][I] ---- Starting application ----
[2023-06-21 00:18:54.697][I] Starting UI
[2023-06-21 00:18:58.482][W] "C:/Users/user/Desktop/test.tif" is not supported. Error: Image format is not supported.
[2023-06-21 00:24:21.297][I] ---- Closing application ----
```

**Comment by Lymphatus** [comment]
Pushed a fix in https://github.com/Lymphatus/caesium-image-compressor/commit/15e18dca8fc160c5784ce36e70e2b68d9b5b3580
Will be available in the next release. Thanks for the feedback.

**Comment by Lymphatus** [comment]
Fixed in [v2.4.1](https://github.com/Lymphatus/caesium-image-compressor/releases/tag/v2.4.1)
